virttest/remote_commander/remote_runner.py

```diff
@@ -295,6 +297,10 @@ def parse_func_name(self, func_name, commander):
         :param func_name: Function name
         :param commander: Where to execute the command (remote or local)
         """
+        for name in func_name:
```

Comment:
@ldoktor does it always require to have two acks to accept pull reqs? Do we have some policy about this requirements?

@xutian or @will-Do could you please give the second ack?

I didn't realise that it required so many efforts to get accepted code.